packages/cli/src/config/extension-manager.ts

```diff
@@ -616,15 +616,21 @@ export class ExtensionManager extends ExtensionLoader {
       throw new Error(`Extension with name ${name} does not exist.`);
     }
 
-    const scopePath =
-      scope === SettingScope.Workspace ? this.workspaceDir : os.homedir();
-    this.extensionEnablementManager.disable(name, true, scopePath);
-    extension.isActive = false;
-    await this.maybeStopExtension(extension);
+    if (scope !== SettingScope.Session) {
```

Comment:
Paranoid: I tend to dislike if statements that match multiple cases with a singular exception.

Can we instead refine this to be inclusive of the situations you want to explicitly handle?

basically: 

```
const INCLUDED_SCOPES = new Set<SettingScope>([
  SESSION,
  DESKTOP,
 WHATEVER,
]);
if(INCLUDED_SCOPES.has(scope))
```

Reply:
Noted and understood.